GOVERNANCE.md

```diff
@@ -0,0 +1,156 @@
+nomination will result in discussion that cannot take more than a month and
+then admission to the SC by consensus. During that time deadlocked votes of the SC will
+be postponed until the new member has joined and another vote can be held.
+
+The Zarr steering council may be contacted at `zarr-steering-council@googlegroups.com`,
+or via the [@zarr-developers/steering-council](https://github.com/orgs/zarr-developers/teams/steering-council) GitHub team.
+
+# Decision Making Process
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the issue trackers of the https://github.com/zarr-developers repositories.
+Occasionally, sensitive discussion may occur via a private message via the GitHub team:
+https://github.com/orgs/zarr-developers/teams/core-devs
+
+Decisions should be made in accordance with the mission and values of the Zarr project.
+
+Zarr uses a “consensus seeking” process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+Core developers are expected to distinguish between fundamental objections to a
+proposal and minor perceived flaws that they can live with, and not hold up the
+decision-making process for the latter.  If no option can be found without
+objections, the decision is escalated to the SC, which will itself use
+consensus seeking to come to a resolution. In the unlikely event that there is
+still a deadlock, the proposal will move forward if it has the support of a
+simple majority of the SC.
+
+Decisions (in addition to adding core developers and SC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction of a
+  sentence, require approval by a core developer *and* no disagreement or requested
+  changes by a core developer on the issue or pull request page (lazy
+  consensus). Core developers are expected to give "reasonable time" after approval and before merging for others
+  to give their opinion on the pull request if they’re not confident others
+  would agree, where "reasonable time" is likely one working day.
+
+- **Code changes and major documentation changes** require agreement by *one*
+  core developer *and* no disagreement or requested changes by a core developer
+  on the issue or pull-request page (lazy consensus). For all changes of this type,
+  core developers are expected to give "reasonable time" after approval and before
+  merging for others to weigh in on the pull request in its final state, where
+  "reasonable time" is likely a few working days.
+
+- **Changes to the Python API principles** require a dedicated issue on our
+  [issue tracker](https://github.com/zarr-developers/zarr-python/issues) and follow the
+  decision-making process outlined above, though "reasonable time" is likely extended
+  to at least a week.
```

Comment:
Should we have a similar section for spec and other repos? Or would it make sense to generalize this to cover all (even non-Python) repos?

Reply:
Something like 15fc40f11a955c67ce001ad671845cfdfc70ad80 ?

Reply:
FWIW I think we'll need a separate discussion to figure out the process for the specs repo, it's a bit different from the code repos. E.g., any changes to specs can impact on multiple implementations, and so will need higher level of consensus. 

How about we:

* Describe a general process that applies to all code repositories in the zarr-developers organisation.
* Say that a separate process applies to the zarr-specs repository, to be defined.

Reply:
@alimanfoo _et al_. : please see d8f82d3377bc8c228b8890e3073744ac90e853cd